<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CModelBrowserComponent</class>
 <widget class="QFrame" name="CModelBrowserComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Model browser</string>
  </property>
  <layout class="QVBoxLayout" name="vl_ModelBrowser" stretch="0,0">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tw_Tab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tb_ModelAndPosition">
      <attribute name="title">
       <string>Model and position</string>
      </attribute>
      <layout class="QVBoxLayout" name="vl_Position">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QGroupBox" name="gb_Model">
         <property name="title">
          <string>Model</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="swift::gui::views::CAircraftModelView" name="tvp_AircraftModels">
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="wi_LoadModels" native="true">
            <layout class="QHBoxLayout" name="hl_LoadModels">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="lbl_ModelSetInfo">
               <property name="text">
                <string>Model set info</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignRight">
              <widget class="QPushButton" name="pb_LoadModelSet">
               <property name="text">
                <string>load model set</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_Parts">
         <property name="title">
          <string>Use</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QCheckBox" name="cb_UseParts">
            <property name="text">
             <string>use parts</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="cb_UsePBH">
            <property name="text">
             <string>use PBH</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="cb_UseCG">
            <property name="text">
             <string>use CG (vert.offset)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_RelativePosition">
         <property name="title">
          <string>Relative position</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="swift::gui::editors::CRelativeAircraftPosition" name="editor_RelativePosition"/>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QPushButton" name="pb_SetRelativePosition">
            <property name="text">
             <string>set</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_AbsolutePosition">
         <property name="title">
          <string>Absolute position</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="swift::gui::editors::CCoordinateForm" name="editor_Coordinate">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tb_PartsAndMisc">
      <attribute name="title">
       <string>PBH/Parts/CG</string>
      </attribute>
      <layout class="QVBoxLayout" name="vl_PartsAndMisc">
       <item>
        <widget class="QGroupBox" name="gb_Pbh">
         <property name="title">
          <string>PBH</string>
         </property>
         <layout class="QVBoxLayout" name="vl_Pbh">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="swift::gui::editors::CPbhsForm" name="editor_Pbh">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Parts</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="swift::gui::editors::CAircraftPartsForm" name="editor_AircraftParts">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>125</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_CG">
         <property name="title">
          <string>CG/vertical offset</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="le_CG"/>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_OverrideCG">
            <property name="text">
             <string>override CG</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wi_DisplayAndInfo" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="le_Info">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_ModelInfo">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_Remove">
        <property name="text">
         <string>remove</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_Display">
        <property name="text">
         <string>display</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::editors::CRelativeAircraftPosition</class>
   <extends>QFrame</extends>
   <header>gui/editors/relativeaircraftposition.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::editors::CCoordinateForm</class>
   <extends>QFrame</extends>
   <header>gui/editors/coordinateform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::views::CAircraftModelView</class>
   <extends>QTableView</extends>
   <header>gui/views/aircraftmodelview.h</header>
  </customwidget>
  <customwidget>
   <class>swift::gui::editors::CAircraftPartsForm</class>
   <extends>QFrame</extends>
   <header>gui/editors/aircraftpartsform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::editors::CPbhsForm</class>
   <extends>QFrame</extends>
   <header>gui/editors/pbhsform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tw_Tab</tabstop>
  <tabstop>tvp_AircraftModels</tabstop>
  <tabstop>pb_LoadModelSet</tabstop>
  <tabstop>cb_UseParts</tabstop>
  <tabstop>cb_UsePBH</tabstop>
  <tabstop>cb_UseCG</tabstop>
  <tabstop>pb_SetRelativePosition</tabstop>
  <tabstop>le_Info</tabstop>
  <tabstop>le_ModelInfo</tabstop>
  <tabstop>pb_Remove</tabstop>
  <tabstop>pb_Display</tabstop>
  <tabstop>le_CG</tabstop>
  <tabstop>cb_OverrideCG</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
